config-controller/pkg/client/client.go

```diff
@@ -211,28 +200,21 @@ func New(ctx context.Context, opts ...clientOptions) (CachedPolicyClient, error)
 	}
 
 	if c.svc == nil {
-		err := retry.WithRetry(func() error {
-			gc, innerErr := newGrpcClient(ctx)
-			if innerErr != nil {
-				log.Error(innerErr, "Failed to connect to Central")
-			}
-
-			c.svc = gc
-
-			if innerErr = c.EnsureFresh(ctx); innerErr != nil {
-				log.Error(innerErr, "Failed to initialize client")
-			}
-
-			return innerErr
-		}, retry.Tries(retryCount), retry.WithExponentialBackoff())
-
+		gc, err := newGrpcClient(ctx)
 		if err != nil {
-			return nil, errors.Wrap(err, "could not initialize policy client")
+			log.Error(err, "Failed to connect to Central")
 		}
-	} else {
+
+		c.svc = gc
+	}
+
+	for {
 		if err := c.EnsureFresh(ctx); err != nil {
 			log.Error(err, "Failed to initialize client")
```

Comment:
nit: to avoid 17 minutes of log entries, should we log this once and then log the success that eventually comes?